src/main/java/io/github/thebusybiscuit/slimefun4/utils/ChatUtils.java

```diff
@@ -80,4 +80,21 @@ public static void awaitInput(@Nonnull Player p, @Nonnull Consumer<String> callb
         return builder.toString();
     }
 
+    /**
+     * This method adds an s to a string if the supplied integer is not 1.
+     *
+     * @param string
+     *      The string to potentially pluralize
+     * @param count
+     *      The amount of things
+     * @return
+     *      {@code string} if {@code count} is 1 else {@code string + "s"}
+     */
+    public static @Nonnull String checkPlurality(@Nonnull String string, int count) {
```

Comment:
test? :)

Reply:
should this throw an ``IllegalArgumentException`` for negative values or is ``-1 bananas`` fine?